installer/pom.xml

```diff
@@ -1,261 +1,265 @@
-<project
```

Comment:
Unnecessary formatting changing make it challenging to determine if there are functional update buried in the noise. Please revert these updates.